massa-execution-exports/src/test_exports/config.rs

```diff
@@ -76,6 +76,7 @@ impl Default for ExecutionConfig {
             broadcast_slot_execution_traces_channel_capacity: 5000,
             max_execution_traces_slot_limit: 320,
             block_dump_folder_path,
+            max_recursive_calls_depth: 50,
```

Comment:
this is set to 25 in constant.rs so maybe we should be consistent here. Unless there is a good reason for not being to?

Reply:
Good catch, changed :)